redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/__tests__/incidentImageCarousel.test.js

```diff
@@ -1,5 +1,18 @@
 import { createElement } from 'lwc';
 import IncidentImageCarousel from 'c/incidentImageCarousel';
+import { registerApexTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+import { getRecord } from 'lightning/uiRecordApi';
+import { getRelatedPictures } from '@salesforce/apex/IncidentController.findRelatedFiles';
+import { registerLdsTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
```

Comment:
To simplify imports this should be added to the import in line 3.

Reply:
removed.